README.md

```diff
@@ -45,7 +44,9 @@ The accessibility column indicates whether the functionality is accessible in it
 | [radio](./packages/radio)                       | [![radio](https://img.shields.io/npm/v/@lion/radio.svg)](https://www.npmjs.com/package/@lion/radio)                                  | Radio from element                                     | ✔️                         |
```

Comment:
I guess we can remove @lion/radio too since it's deprecated like checkbox

Reply:
I missed this one ;)